OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -250,7 +284,7 @@ public void onSuccess(@NonNull AuthResponse result) {
                 @Override
                 public void onException(Exception ex) {
                     if (ex instanceof HttpStatusException &&
-                            ((HttpStatusException) ex).getStatusCode() == HttpStatus.UNAUTHORIZED) {
+                            ((HttpStatusException) ex).getStatusCode() == HttpStatus.BAD_REQUEST) {
```

Comment:
is there any use case of getting `UNAUTHORIZED` response code.

Reply:
@omerhabib26 just this question is still pendding.

Reply:
@farhan-arshad-dev currently on iOS & Android we are only handling *BAD_REQUEST* response code.